lfs/util.go

```diff
@@ -173,6 +173,40 @@ func (p *currentToRepoPathConverter) Convert(filename string) string {
 	return filepath.ToSlash(reltoroot)
 }
 
+// Convert filenames expressed relative to the current directory to be
+// relative to the repo root. Useful when calling git with arguments that requires them
+// to be rooted but the user is in a subdir of their repo & expects to use relative args
+// Pass in a channel which you will fill with relative files & receive a channel which will get results
+func NewCurrentToRepoPatternConverter(cfg *config.Configuration) (PathConverter, error) {
```

Comment:
I originally implemented this as an additional method (`ConvertPath`), but then realized it's an interface.  I think I'd like to keep this as it is, since I think there's value in having one implementation undo the other one, and then having this as a special case.  I can imagine us needing to use the wrapped implementation in the future, so I'd like to preserve it.

I'll update the comment.

Reply:
Sure, makes sense, and thanks for tuning the comment.